src/views/v2/list/nav/next.php

```diff
@@ -21,14 +21,12 @@
 		rel="next"
 		class="tribe-events-c-nav__next tribe-common-b2 tribe-common-b1--min-medium"
 		data-js="tribe-events-view-link"
-	>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__next-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
+	><?php
+			$events_label = '<span class="tribe-events-c-nav__next-label-plural">' . tribe_get_event_label_plural() . '</span>';
 			echo wp_kses(
 				/* translators: %s: Event (plural or singular). */
 				sprintf( __( 'Next %1$s', 'the-events-calendar' ), $events_label ),
 				[ 'span' => [ 'class' => [] ] ]
 			);
-		?>
-	</a>
+	?></a>
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 10648b1ebe7bdb97729aa367fb2cbe90</sub>